mne/annotations.py

```diff
@@ -817,10 +817,18 @@ def _select_annotations_based_on_description(descriptions, event_id, regexp):
 
 
 def _check_event_id(event_id, raw):
+    from mne.io.brainvision.brainvision import _BVEventParser
+    from mne.io.brainvision.brainvision import _check_bv_annot
+
     if event_id is None:
         return _DefaultEventParser()
     elif event_id == 'auto':
-        return getattr(raw, '_get_auto_event_id', _DefaultEventParser)()
+        candidate = getattr(raw, '_get_auto_event_id', _DefaultEventParser)
```

Comment:
I don't think that using a `@classmethod` as a dispatcher makes sense anymore.

@larsoner I would like to see your comments on that. 
('cos I guess that if we get rid of this we'll have to deprecate the class method don't we?)

Reply:
the class method is private, no need for dep cycle, feel free to simplify